### ISSUE TITLE
Wanted to understand the procedure to add to the paper

### ISSUE THREAD

**Issue by sumukhshetty** [Mechanism-Labs/Summary-of-Knowledge-SoK-Scalability-of-Blockchains#1]
I understand that i pull the master and edit on a new branch.
But methodologies and frameworks differ, so wanted to know if you've set some sort of standard.
I'm currently working on writing a post that distinguishes between Plasma, Raiden, and Casper(Pos + Sharding).
Those 4 concepts go by different names in different projects, so what would the right way to go about it be?

**Comment by MaazUddin11** [comment]
Hey Sumukh, thanks for getting involved in the process!

I see what you mean and how differences might arise. We haven't set a standard just yet, but what ideas did you have for these 4 concepts? I'd love to hear your opinion on this

**Comment by sumukhshetty** [comment]
Nothing specific. I think one way to do this is getting the main topic and create subheadings to name all the implementations in different networks.
Example: Lightning and Raiden could be called p2p state channel networks but I think this may confuse people even more. If we want to avoid bias, this would be the best route though.

I'm not sure if there are other alternatives for Plasma, but plasma by itself is split into multiple different implementations like Plasma MVP, Plasma Cash, Plasma XT. 
PoS is straightforward . But implementations vary, so we may have to add every implementation we come across under the subsection. Instead of just doing Casper.
Same goes for sharding, we could talk about zilliqa and ethereum here from what I know.

 I'm not aware if there are any generic terms for raiden or lightning. Maybe we can put them under implementations in state channels. Not sure if that's too broad though. Plasma may also come under state channels too

**Comment by MaazUddin11** [comment]
Yeah that's pretty similar to what we were thinking. We got our inspiration from this [SoK on Consensus](https://arxiv.org/pdf/1711.03936.pdf). So we were thinking of breaking the whole thing down by where in the [blockchain stack](https://medium.com/mechanism-labs/the-blockchain-stack-ca953c088692) each solution would lie and then go over the solution and give some technical detail like the SoK mentioned above does, but not necessarily dive too deep. 

The idea is to give readers a good high level understanding of each solution, but also to point them to more readings so they can formulate opinions of their own. Take PoS as an example. We would give an overview of how PoS can scale better than PoW and describe the differences between the two (voting, staking, slashing etc.). But then we could briefly touch on how Tendermint does PoS different than Casper and different than Algorand etc. and cite different relevant sources that could help people understand how each one is built/implemented and potential tradeoffs each makes.

**Comment by sumukhshetty** [comment]
Awesome. How do you guys go about drafting these SoK's. I don't want it to be a case of too many cooks spoiling the broth. Maybe I'll write out the post on raiden, pos, and plasma first and then add that info here.

**Comment by MaazUddin11** [comment]
So far we don't have a formal process or anything for that. If you want to work on a specific part you think you know well, then by all means go for it! We want as much community engagement as possible. So feel free to add what you know and submit a pull-request. And also ask questions like these whenever you need

**Comment by MaazUddin11** [comment]
Hey Sumukh, if you are still interested in being involved email me (maaz@mechanismlabs.io). We have gathered a few people who all want to contribute and I can get you up to speed on how to move forward with this if you are interested.
